### ISSUE TITLE
Applying ingest stylesheet to dublin core with two dates fails

### ISSUE THREAD

**Issue by hsilva-keep** [keeps/roda#242]
And the XML document is valid, when validating it against XML schema.
```
Error on line 41 
  XPTY0004: A sequence of more than one item is not allowed as the @select attribute of
  analyze-string ("2016-03-09", "2016-03-09") 
  at xsl:apply-templates (#10)
     processing /simpledc
```

**Comment by sleroux-keep** [comment]
Fixed with c44c25c1175dce73ab7a2e6ce5fe5458e793a6b1
for the input
<?xml version="1.0" encoding="UTF-8"?>
<simpledc xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <title>TITLE4</title>
  <title>TITLE5</title>
  <creator>CREATOR1</creator>
  <creator>CREATOR2</creator>
  <description>DESC1</description>
  <description>DESC2</description>
  <publisher></publisher>
  <identifier></identifier>
  <date>2016-03-14</date>
  <date>2017-03-15</date>
  <date>2017-03-16</date>
</simpledc>

The date 2016-03-14 goes to the dateInitial field
The date 2017-03-15 goest to the dateFinal field
All the three dates goes to the date_txt field (so it is possible to search all the dates in a single field)
/(same behavior for the title and the description)

**Comment by sleroux-keep** [comment]
If the "_txt" mappings are not necessary, assign the ticket back to me.
